ci_scripts/flake8_diff.sh

```diff
@@ -125,7 +125,7 @@ check_files() {
     if [ -n "$files" ]; then
         # Conservative approach: diff without context (--unified=0) so that code
         # that was not changed does not create failures
-        git diff --unified=0 $COMMIT_RANGE -- $files | flake8 --diff --show-source $options
+        git diff --unified=0 $COMMIT_RANGE -- $files | flake8 --ignore E402 --diff --show-source $options
```

Comment:
Just asking, what exactly is this additional ignore?

Reply:
https://lintlyci.github.io/Flake8Rules/rules/E402.html